charts/scalardb-mcp-server/templates/scalardb-mcp-server/ingress.yaml

```diff
@@ -0,0 +1,62 @@
+{{- if .Values.ingress.enabled -}}
+{{- $fullName := include "scalardb-mcp-server.fullname" . -}}
```

Comment:
Just a question.

It seems that there is no condition for this variable. In other words, the value of `$fullName` always relies on the value of `scalardb-mcp-server.fullname`.

I'd like to confirm the reason why we need to use the variable here. What is the reason why you use the variable here?